Makefile.gen

```diff
@@ -24,10 +24,18 @@ symlink-gopath:
 	@[[ $(shell pwd) == $(shell realpath $(CANONICAL_GOPATH)) ]] \
 		|| (echo "expected $(shell pwd) to equal $(shell realpath $(CANONICAL_GOPATH))" ; exit 1)
 
+CLIENTGEN_TOOLS := client-gen deepcopy-gen informer-gen lister-gen conversion-gen
+# This should match the k8s.io/code-generator version in go.mod.
+CLIENTGEN_VERSION := "v0.22.2"
+
+.PHONY: $(CLIENTGEN_TOOLS)
+$(CLIENTGEN_TOOLS):
```

Comment:
hmm. i guess this is ok. kinda weird to have a target with spaces, but if it's phony i guess it's ok. 

maybe add an install-clientgen-tools and clean-client-gen-tools too just for consistency with the other install targets.

Reply:
done